### PR TITLE
Content updated

### PR DESCRIPTION
**Pull request by fulyauluturk**
Integrations guide content updated and aligned with how SC behaves. Added missing integrations.

### CODE REVIEW

File: docs/guide/integrations-guide.md [hunk 1]
```diff
@@ -99,38 +112,54 @@ After you install and activate an integration, you can:
...
 New integration creation userflow is the same for both monitoring as well as logging App. While Apache monitoring App is being created you can also create complementary logs App as well. It makes perfect sense, and it is the only way to have full observability into performance and issues of the Apache server for example, and that particular piece of a IT infrastructure puzzle.
 
-In other words, as monitoring App is being created you can create a logging App as well and vice versa. You can also invite other devops team members and provide them with the same tools and insights while using RBAC (role based access control) model to restrict or provide proper administrative user rights.
+In other words, as monitoring App is being created you can create a logging App as well and vice versa. 
+
+You can connect Logs and Monitoring Apps for quick access later when you are troubleshooting.
+
+You can also invite other devops team members and provide them with the same tools and insights while using RBAC (role based access control) model to restrict or provide proper administrative user rights.
+
+### Connected Apps
```
Comment (otisg) [inline docs/guide/integrations-guide.md:141]: @fulyauluturk JFYI we probably want to extract this into its own page with screenshots, explanation, linking to/from Split View, etc.

File: docs/guide/integrations-guide.md [hunk 2]
```diff
@@ -99,38 +112,54 @@ After you install and activate an integration, you can:
 - [Twilio](../integration/alerts-twilio-integration/)
 - [VictorOps](../integration/alerts-victorops-integration/)
 - [Zapier](../integration/alerts-zapier-integration/)
+- [Signl4](../integration/alerts-signl4-integration/)
+- [GoogleChat](../integration/alerts-googlechat-integration/)
+- [AlertOps](../integration/alerts-alertops-integration/)
+- [ServiceNow](../integration/alerts-servicenow-integration/)
```
Comment (cpipilas) [inline docs/guide/integrations-guide.md:118]: may I suggest to keep the sorting in alphabetical order?

**Review by cpipilas — COMMENTED** [review]
LGTM, minor suggestions added

File: docs/guide/integrations-guide.md [hunk 3]
```diff
@@ -99,38 +112,54 @@ After you install and activate an integration, you can:
...
+In other words, as monitoring App is being created you can create a logging App as well and vice versa. 
+
+You can connect Logs and Monitoring Apps for quick access later when you are troubleshooting.
+
+You can also invite other devops team members and provide them with the same tools and insights while using RBAC (role based access control) model to restrict or provide proper administrative user rights.
+
+### Connected Apps
+
+Connecting a Monitoring App to a Logs App will pre-select that Logs App when you decide to correlate metrics. When you
+receive an alert notification for an App, the alert notification will automatically charts from
```
[the remark this answers is not in the document]
Comment (fulyauluturk) [reply docs/guide/integrations-guide.md:144]: Changed it to > will contain charts from connected Apps

File: docs/guide/integrations-guide.md (hunk 1 above)
Comment (fulyauluturk) [reply docs/guide/integrations-guide.md:142]: Ticket created https://sematext.atlassian.net/browse/SC-14667

**Pull request merged**